qdrant-landing/content/documentation/tutorials/retrieval-quality.md

```diff
@@ -0,0 +1,231 @@
+are based on the number of relevant documents in the top-k search results. Others, such as [Mean Reciprocal Rank (MRR)](https://en.wikipedia.org/wiki/Mean_reciprocal_rank), 
+take into account the position of the first relevant document in the search results. DCG and NDCG metrics are, in turn, based on the relevance 
+score of the documents.
+
+If we treat the search pipeline as a whole, we could use them all. The same is true for the embeddings quality evaluation. However, for the 
+ANN algorithm itself, anything based on the relevance score or ranking is not applicable. Ranking in vector search relies on the distance
+between the query and the document in the vector space, however distance is not going to change due to approximation, as the function is
+still the same. 
+
+Therefore, it only makes sense to measure the quality of the ANN algorithm by the number of relevant documents in the top-k search results, 
+such as `precision@k`. It is calculated as the number of relevant documents in the top-k search results divided by `k`. In case of testing
+just the ANN algorithm, we can use the exact kNN search as a ground truth, with `k` being fixed. It will be a measure on **how well the ANN
+algorithm approximates the exact search**.
+
+## Measure the quality of the search results
+
+Let's build a quality evaluation of the ANN algorithm in Qdrant. We will, first, call the search endpoint in a standard way to obtain
+the approximate search results. Then, we will call the exact search endpoint to obtain the exact matches, and finally compare both results
+in terms of precision.
+
+Before we start, let's create a collection, fill it with some data and then start our evaluation. We will use the same dataset as in the
+[Loading a dataset from Hugging Face hub](/documentation/tutorials/huggingface-datasets/) tutorial, `Qdrant/arxiv-titles-instructorxl-embeddings`
+from the [Hugging Face hub](https://huggingface.co/datasets/Qdrant/arxiv-titles-instructorxl-embeddings). Let's download it in a streaming
+mode, as we are only going to use part of it.
+
+```python
+from datasets import load_dataset
+
+dataset = load_dataset(
+    "Qdrant/arxiv-titles-instructorxl-embeddings", split="train", streaming=True
+)
+```
+
+We need some data to be indexed and another set for the testing purposes. Let's get the first 10000 items for the training and the next 1000
+for the testing.
+
+```python
+dataset_iterator = iter(dataset)
+train_dataset = [next(dataset_iterator) for _ in range(10000)]
+test_dataset = [next(dataset_iterator) for _ in range(1000)]
+```
+
+Now, let's create a collection and index the training data. This collection won't be created with the default configuration, as we want to
+ensure the approximate search is performed. Therefore, we will set the `indexing_threshold` to `1`, which means that the indexing will be
```

Comment:
'indexing threshold' isn't too clear for a concept plus no references are found on the indexing page (which is provided as the reference) .

Reply:
I would agree, that setting `indexing_threshold: 1` is usually not the best practice. Optimizer will be triggered on each insertion.

I would propose to just use default value